linera-base/src/crypto/mod.rs

```diff
@@ -213,6 +229,58 @@ impl TryFrom<&[u8]> for AccountSignature {
     }
 }
 
+/// A trait for signing keys.
+#[cfg_attr(not(web), async_trait)]
+#[cfg_attr(web, async_trait(?Send))]
+pub trait SigningKey {
```

Comment:
I do suggest that this be named `Signer` or some other name that doesn't imply that it actually contains a key.

Reply:
Renamed.